sdk/nodejs/dagger/dist/engine.js

```diff
@@ -20,10 +20,10 @@ export class Engine {
             const args = ["dev"];
             // exit with error if we are not using the non-Cloak dagger binary (< 0.3.0)
             try {
-                execaCommandSync("dagger dev --help");
+                execaCommandSync("cloak dev --help");
             }
             catch (err) {
-                console.error("⚠️  Please ensure that dagger binary in $PATH is v0.3.0 or newer - a.k.a. Cloak");
+                console.error("⚠️  Please ensure that cloak binary in $PATH is v0.3.0 or newer - a.k.a. Cloak");
```

Comment:
We missed this `- a.k.a. Cloak`

Reply:
Thx for :eyes: . Should be fixed here: https://github.com/dagger/dagger/pull/3576